Resources/doc/reference/form_types.rst

```diff
@@ -370,6 +370,9 @@ According the choice made only associated fields are displayed. The others field
 map
   Associative array. Describes the fields that are displayed for each choice.
 
+**TIP**: To mask a subfield replace `.` by `__`.
```

Comment:
Remember the exemple with the Engine and their characteristics.

```
// EngineAdmin.php
protected function configureFormFields(FormMapper $formMapper) 
{      
  $formMapper->add('type', ChoiceFieldMaskType::class, [
          'required' => true,
          'choices' => [
             'car' => 'car',
             'bike' => 'bike'
           ],
          'map' => [
            'car' => ['characteristic.puissance']
          ],
      ])
      ->add('characteristic.puissance', 'text')
}
```

This will not work because the `characteristic.puissance` input is renamed as `characteristic__puissance` in the html so to make this work you have to write it like that 

```
'map' => [
            'car' => ['characteristic__puissance']
]
```